README.md

```diff
@@ -59,107 +59,117 @@ Click the data migrator to launch it. You will be prompted for a connection to D
 
-1.	Select **Generate/Modify Export Config** from the Schema Config tab of the data migrator and then select **Next**
- ![Export Config Start](images/ExportConfigStart.png "Export Config Start")
+1. Select **Data Export** from the top ribbon which will bring you to the below page:
+ ![Initial Export Page](images/InitialExportPage.png "Initial Export Page")
 
-2.	Select the entities and the attributes and then in the **Export Config** browse to a location and input a File Name (in this example we are creating a xml file for Accounts & Contacts), then select **Save**
- ![Export Config Select Entity](images/ExportConfigSelectEntity.png "Export Config Select Entity")
+2. If you have an exisiting export config file, you can click on 'Load' to edit that file. This will pre-input the configs that you had stored in that file.
+![Load Export Config file](images/LoadExportConfig.png "Load Export Config file")
 
-3.	Save the Export Config
- ![Export Config Save](images/ExportConfigSave.png "Export Config Save")
+3. Add in (or edit if you have loaded an export config file) Fetch and Write Settings and ensure you add a schema file.
+ ![Edited Export Config file](images/EditedExportConfigs.png "Edited Export Config file")
 
-4.	Once the export config is generated, a "Successfully created XML file" dialog will pop up and the export JSON file will be generated at the specified location. Below is an example of the generated file
- ![Export Config Output](images/ExportConfigOutput.png "Export Config Output")
+4. To save the new or updated export config file, click on 'Save'.
+ ![Save Export Configs](images/SaveExportConfigs.png "Save Export Configs")
+
+Your export configs file will be saved at the specified location.
+
+Below shows an example export config file:
+
+ ![Export Config Example](images/ExportConfigExample.png "Export Config Example")
 
 **NB:** Ensure the JsonFolderPath exists
 
 ### Import Config File
 The CDS data migrator import user interface exposes only a subset of the available configuration points for the data migration operation. Through the provision of an import configuration file, the user can have a fine grain control of the import process. When the import config is not specified then the default settings are applied. 
 For more on the data migration config settings [see](https://github.com/Capgemini/xrm-datamigration#Usage)
 
-1.	Select **Generate/Modify Import Config** from the Schema Config tab of the data migrator and then select **Next**
- ![Import Config Start](images/ImportConfigStart.png "Import Config Start")
+1. Select **Data Import** from the top ribbon which will bring you to the below page:
+ ![Initial Export Page](images/InitialImportPage.png "Initial Export Page")
+
+2. If you have an exisiting import config file, you can click on 'Load' to edit that file. This will pre-input the configs that you had stored in that file.
```

Comment:
```suggestion
2. If you have an existing import config file, you can click on **Load** to edit that file. This will pre-input the configs that you had stored in that file.
```